rocketpy/Rocket.py

```diff
@@ -477,6 +482,11 @@ def addSurfaces(self, surfaces, positions):
             regarding the rocket's coordinate system.
             If a list is passed, it will correspond to the position of each item
             in the surfaces list.
+            For NoseCone type, position is relative to the nose cone tip.
+            For Fins type, position is relative to the point belonging to
+            the root chord which is highest in the rocket coordinate system.
```

Comment:
Does the word "highest" here consider that the coordinate system can be defined as "nozzle-to-ogive" or "ogive-to-nozzle" ?

Reply:
Yes, the csys variation is already being considered